include/rtc.h

```diff
@@ -0,0 +1,57 @@
+/**
+ * @file rtc.h
+ * @brief Real-Time Clock Subsystem
+ * @ingroup rtc
+ */
+#ifndef __LIBDRAGON_RTC_H
+#define __LIBDRAGON_RTC_H
+
+#include <stdint.h>
+#include <stdbool.h>
+
+/**
+ * @addtogroup rtc
+ * @{
+ */
+
+/**
+ * @brief Structure for storing RTC time data.
+ */
+typedef struct rtc_time_t
+{
+    /** @brief Year. [1900-20XX] */
+    uint16_t year;
+    /** @brief Month. [0-11] */
+    uint8_t month;
+    /** @brief Day of month. [1-31] */
+    uint8_t day;
+    /** @brief Hours. [0-23] */
+    uint8_t hour;
+    /** @brief Minutes. [0-59] */
+    uint8_t min;
+    /** @brief Seconds. [0-59] */
+    uint8_t sec;
+    /** @brief Day of week. [0-6] (Sun-Sat) */
+    uint8_t week_day;
+} rtc_time_t;
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+bool joybus_rtc_is_present( void );
+bool joybus_rtc_is_busy( void );
+bool joybus_rtc_is_writable( void );
+
+void joybus_rtc_run( void );
+void joybus_rtc_set( const rtc_time_t * rtc_time );
+
+void joybus_rtc_read_time( rtc_time_t * rtc_time );
```

Comment:
I think this should be symmetric with the set command. Either set/get or read/write.

Reply:
Addressed in https://github.com/DragonMinded/libdragon/pull/152/commits/e46fbb07a6cbb01a98f05a2d3b170d4ff1860b77